tests/lib/mcp/mcp-server.js

```diff
@@ -55,6 +56,14 @@ describe("MCP Server", () => {
 		// Note: must connect server first or else client hangs
 		await mcpServer.connect(serverTransport);
 		await client.connect(clientTransport);
+
+		sinon
+			.stub(process, "emitWarning")
+			.withArgs(sinon.match.any, "ESLintIgnoreWarning");
+	});
+
+	afterEach(async () => {
```

Comment:
```suggestion
	afterEach(() => {
```

No need for `async` here?